plugins/vppv2/l3plugin/descriptor/static_route.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package descriptor
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/pkg/errors"
+
+	"github.com/ligato/cn-infra/logging"
+	scheduler "github.com/ligato/vpp-agent/plugins/kvscheduler/api"
+	ifdescriptor "github.com/ligato/vpp-agent/plugins/vppv2/ifplugin/descriptor"
+	"github.com/ligato/vpp-agent/plugins/vppv2/l3plugin/descriptor/adapter"
+	"github.com/ligato/vpp-agent/plugins/vppv2/l3plugin/vppcalls"
+	"github.com/ligato/vpp-agent/plugins/vppv2/model/interfaces"
+	"github.com/ligato/vpp-agent/plugins/vppv2/model/l3"
+)
+
+const (
+	// StaticRouteDescriptorName is the name of the descriptor for static routes.
+	StaticRouteDescriptorName = "static-route"
+
+	// dependency labels
+	routeOutInterfaceDep = "interface-exists"
+)
+
+// RouteDescriptor teaches KVScheduler how to configure Linux routes.
+type RouteDescriptor struct {
```

Comment:
There are couple of comments that mention *Linux* as opposed to VPP, but I guess this is because the PR is still WIP.